tests/dummy/app/app.js

```diff
@@ -1,13 +1,14 @@
 import Ember from 'ember'
+const {Application} = Ember
+import config from './config/environment'
 import Resolver from './resolver'
 import loadInitializers from 'ember-load-initializers'
```

Comment:
I am similarly surprised by this. I hadn't noticed that.  How strange. I know I did a linting pass to satisfy the sort order warnings, but I might have added this afterward (?).  I'll double check it, as I agree that I'd prefer the local ones separated.